<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="powerButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(85, 87, 83);
image: url(:/images/power-button.png);
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #30D5C8;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="intensityUpButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>250</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(85, 87, 83);
image: url(:/images/up-arrow.png);
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #30D5C8;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="endButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>480</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 63 11pt &quot;URW Gothic&quot;;
color: rgb(0, 0, 0);
	background-color: rgb(247, 215, 255);
}

QPushButton:pressed {
	background-color: #30D5C8;
}</string>
    </property>
    <property name="text">
     <string>End Session</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="deviceGraphic">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>811</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/device.png);
background-color: rgba(255, 255, 255, 0);
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="intensityDownButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(85, 87, 83);
image: url(:/images/down-arrow.png);
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #30D5C8;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="powerLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic</family>
      <pointsize>11</pointsize>
      <weight>7</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(85, 87, 83);
	font: 63 11pt &quot;URW Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>On/Off</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="intensityLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>290</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic</family>
      <pointsize>11</pointsize>
      <weight>7</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(85, 87, 83);
	font: 63 11pt &quot;URW Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Intensity </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>170</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic</family>
      <pointsize>13</pointsize>
      <weight>7</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(223, 190, 235);
font: 63 13pt &quot;URW Gothic&quot;;
background-color: rgb(85, 87, 83);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; font-style:italic;&quot;&gt;OASIS PRO&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="sessionWidget" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>321</width>
      <height>44</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 87, 83);</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="sessionLayout">
       <item>
        <widget class="QPushButton" name="alphaButton">
         <property name="styleSheet">
          <string notr="true">QPushButton{background-color: rgb(255, 255, 255);
	font: 63 11pt &quot;URW Gothic&quot;;}

QPushButton:pressed {
	background-color: rgb(252, 175, 62);
}

</string>
         </property>
         <property name="text">
          <string>Alpha</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="betaButton">
         <property name="styleSheet">
          <string notr="true">QPushButton{background-color: rgb(255, 255, 255);
	font: 63 11pt &quot;URW Gothic&quot;;}

QPushButton:pressed {
	background-color: rgb(252, 233, 79);
}</string>
         </property>
         <property name="text">
          <string>Beta</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deltaButton">
         <property name="styleSheet">
          <string notr="true">QPushButton{background-color: rgb(255, 255, 255);
	font: 63 11pt &quot;URW Gothic&quot;;}

QPushButton:pressed {
	background-color: rgb(138, 226, 52);
}</string>
         </property>
         <property name="text">
          <string>Delta</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="thetaButton">
         <property name="styleSheet">
          <string notr="true">QPushButton{background-color: rgb(255, 255, 255);
	font: 63 11pt &quot;URW Gothic&quot;;}

QPushButton:pressed {
	background-color: rgb(58, 157, 255);
}</string>
         </property>
         <property name="text">
          <string>Theta</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="skinButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>370</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 63 11pt &quot;URW Gothic&quot;;
	color: rgb(0, 0, 0);
	background-color: rgb(247, 215, 255);
}

QPushButton:pressed {
	background-color: #30D5C8;
}</string>
    </property>
    <property name="text">
     <string>Apply To skin</string>
    </property>
   </widget>
   <widget class="QLabel" name="batteryLabel">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>170</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 87, 83);
color: rgb(255, 255, 255);
	font: 63 11pt &quot;URW Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Battery Level:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="batteryValLabel">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>170</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 87, 83);
color: rgb(255, 255, 255);
	font: 63 11pt &quot;URW Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>100</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>370</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(85, 87, 83);
image: url(:/images/check-mark.svg);
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #30D5C8;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="intensityList">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>31</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 87, 83);

QListWidget{ border-color: rgb(85, 87, 83);}

QListWidget:item:selected {  selection-background-color: rgb(255, 255, 255);
selection-color: rgb(255, 255, 255);

background-color: rgb(255, 255, 255);
}

</string>
    </property>
    <property name="flow">
     <enum>QListView::TopToBottom</enum>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
     <property name="font">
      <font>
       <family>URW Gothic</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>252</red>
        <green>233</green>
        <blue>79</blue>
       </color>
      </brush>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
     <property name="font">
      <font>
       <family>URW Gothic</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>252</red>
        <green>233</green>
        <blue>79</blue>
       </color>
      </brush>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
     <property name="font">
      <font>
       <family>URW Gothic</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>252</red>
        <green>233</green>
        <blue>79</blue>
       </color>
      </brush>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
     <property name="font">
      <font>
       <family>URW Gothic</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>245</red>
        <green>121</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
     <property name="font">
      <font>
       <family>URW Gothic</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>245</red>
        <green>121</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
     <property name="font">
      <font>
       <family>URW Gothic</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>245</red>
        <green>121</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
     <property name="font">
      <font>
       <family>URW Gothic</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>239</red>
        <green>41</green>
        <blue>41</blue>
       </color>
      </brush>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
     <property name="font">
      <font>
       <family>URW Gothic</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>204</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>239</red>
        <green>41</green>
        <blue>41</blue>
       </color>
      </brush>
     </property>
    </item>
   </widget>
   <widget class="QListWidget" name="recordList">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>220</y>
      <width>191</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="recordUpButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>230</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(85, 87, 83);
image: url(:/images/up-arrow.png);
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #30D5C8;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="recordDownButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>260</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(85, 87, 83);
image: url(:/images/down-arrow.png);
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #30D5C8;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="powerLED">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>40</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="timerLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>430</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	font: 63 11pt &quot;URW Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="recordYesButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>390</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Yes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="recordNoButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>390</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>No</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>781</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;</string>
    </property>
    <property name="title">
     <string>Device Configuration</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>103</width>
       <height>77</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: none;</string>
        </property>
        <property name="text">
         <string>Time Scale</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="realtimeRadioButton">
        <property name="styleSheet">
         <string notr="true">background-color: none;</string>
        </property>
        <property name="text">
         <string>Real Time</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="relativetimeRadioButton">
        <property name="styleSheet">
         <string notr="true">background-color: none;</string>
        </property>
        <property name="text">
         <string>Relative Time</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="clearRecordsButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>30</y>
       <width>181</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear All Records</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rechargeBatteryButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>60</y>
       <width>181</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Recharge Battery</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>31</y>
       <width>221</width>
       <height>69</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Electrodes Satus</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Electrodes on Skin :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="electrodeStatusDisplay">
          <property name="text">
           <string>False</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Last Connection Test Result:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="connectStrengthDisplay">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="rechargeWarnLabel">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>90</y>
       <width>361</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(239, 41, 41)</string>
     </property>
     <property name="text">
      <string>Warning: Cannot Recharge Battery While a Session is Running!</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="userDesignatedWidget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>251</x>
      <y>250</y>
      <width>321</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Custom Time</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="userInput"/>
     </item>
     <item>
      <widget class="QPushButton" name="customTimeSubmit">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(78, 154, 6);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>SUBMIT</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="groupWidget" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>321</width>
      <height>42</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="group1Button">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	font: 63 11pt &quot;URW Gothic&quot;;
	
}

QPushButton {
	font: 63 11pt &quot;URW Gothic&quot;;
	
}

QPushButton:pressed {
	background-color: #FF0000;
	
}

</string>
       </property>
       <property name="text">
        <string>20 min</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="group2Button">
       <property name="styleSheet">
        <string notr="true">QPushButton:pressed {
	background-color: #FFA500;
	font: 63 11pt &quot;URW Gothic&quot;;
}

QPushButton {
	font: 63 11pt &quot;URW Gothic&quot;;
	
}</string>
       </property>
       <property name="text">
        <string>45 min</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="group3Button">
       <property name="styleSheet">
        <string notr="true">QPushButton:pressed {
	background-color: #FFFF00;
	font: 63 11pt &quot;URW Gothic&quot;;
}

QPushButton {
	font: 63 11pt &quot;URW Gothic&quot;;
	
}</string>
       </property>
       <property name="text">
        <string>User Designed</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="connectionTestLabel">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>480</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Connection Test is Running</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>deviceGraphic</zorder>
   <zorder>powerButton</zorder>
   <zorder>intensityUpButton</zorder>
   <zorder>endButton</zorder>
   <zorder>intensityDownButton</zorder>
   <zorder>powerLabel</zorder>
   <zorder>intensityLabel</zorder>
   <zorder>titleLabel</zorder>
   <zorder>sessionWidget</zorder>
   <zorder>skinButton</zorder>
   <zorder>batteryLabel</zorder>
   <zorder>batteryValLabel</zorder>
   <zorder>startButton</zorder>
   <zorder>intensityList</zorder>
   <zorder>recordList</zorder>
   <zorder>recordUpButton</zorder>
   <zorder>recordDownButton</zorder>
   <zorder>powerLED</zorder>
   <zorder>timerLabel</zorder>
   <zorder>recordYesButton</zorder>
   <zorder>recordNoButton</zorder>
   <zorder>groupBox</zorder>
   <zorder>userDesignatedWidget</zorder>
   <zorder>groupWidget</zorder>
   <zorder>connectionTestLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>908</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOasis_Pro_Simulation">
    <property name="title">
     <string>Oasis Pro Simulation</string>
    </property>
   </widget>
   <addaction name="menuOasis_Pro_Simulation"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
